server/apps/io/dpa.py

```diff
@@ -55,3 +55,16 @@ def _update(self, provider):
             except Exception as ex:
                 self.move_file(self.path, filename, provider=provider, success=False)
                 raise ParserError.parseFileError('DPA', filename, ex, provider)
+
+    def parse_file(self, filename, provider):
```

Comment:
A basic docstring would be useful here, e.g. is `filename` a full or a relative path, etc.